### PR TITLE
Remove "required" values no longer required for alpha deadline

### LINKED ISSUE #787: Add example payload from test data feed
**Is your feature request related to a problem? Please describe.**
We now have example data from Ares, we should put an example in the application and update the schema

**Describe the solution you'd like**
Add an up-to-date optimo/ares payload into the application  and update the schema definition. 

**Describe alternatives you've considered**
N/A

**Testing notes**
N/A

### LINKED ISSUE #795: Align fixture data with ares payload
**Is your feature request related to a problem? Please describe.**
Following the [analysis](https://github.com/BBC-News/simorgh/issues/792) we should align the fixture data with the ares payload.

**Describe the solution you'd like**
Using the examples in https://www.diffchecker.com/3VLSTaBY Add the following values:
`language`, `options`, `relatedContent` block and `analyticsLabels`
Also remove:
`category`, `genre` and `created`

Align the application (propTypes and schema) with the Ares payload, this requires removing the incorrect "required" values in the payload. 

**Describe alternatives you've considered**
N/A

**Testing notes**
Data validator passing is sufficient testing

- [x] Intially labelled with ["Refinement needed"](https://github.com/BBC-News/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by pjlee11**
Resolves #795

This PR compared the previous business logic for what was required in the Ares payload as this has since changed for the alpha deadline. 

1. Updates the schema to represent current business logic supported by Optimo (EG: Optimo doesn’t have seoHeadline, summary, promoHeadline or category yet)
2. Updates the application propTypes to match what is no longer required
~3. Default the MetaData title to `''` due to Optimo not being able to publish the seoHeadline at this time~ - handled in #817
~4. Adds the Ares payload equivalents of scenario's 25 and 27~ - handled in #818 
5. Adds the new schema block `aresMedia`

- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by pjlee11** [comment]
This PR adds `alt-text` as a schema validation however this is a bug and will be changing to `altText` to be consistent with other naming

**Review by jtart — APPROVED** [review]

File: data/schema.yaml [hunk 1]
```diff
@@ -221,6 +242,7 @@ components:
           - $ref: '#/components/schemas/subheadline'
           - $ref: '#/components/schemas/text'
           - $ref: '#/components/schemas/video'
+          - $ref: '#/components/schemas/aresMedia'
```
Comment (BogdanDogaru) [inline data/schema.yaml:247]: Can we change this to just say `media`? I think adding `ares` might confuse things and `media` is generic enough.

**Review by BogdanDogaru — CHANGES_REQUESTED** [review]

File: data/schema.yaml (hunk 1 above)
Comment (pjlee11) [reply data/schema.yaml:247]: _"Ideally Ares shouldn’t be creating `media` blocks unless we know that Optimo won’t also in the future create `media` blocks. So we currently decided to namespace them with who is responsible for doing it.
There is a conversation around a system whereby other people can submit new block types to Optimo without creating name clashes to hopefully resolve this"_

Comment (BogdanDogaru) [reply data/schema.yaml:247]: Thanks for the clarification! PR looks good to me :)

**Review by BogdanDogaru — APPROVED** [review]

**Pull request merged**